comps/comps-foreman-el8.xml

```diff
@@ -410,8 +410,8 @@
       <packagereq type="default">rubygem-gitlab-sidekiq-fetcher-doc</packagereq>
       <packagereq type="default">rubygem-globalid-doc</packagereq>
       <packagereq type="default">rubygem-google-api-client-doc</packagereq>
-      <packagereq type="default">rubygem-google-cloud-env-doc</packagereq>
       <packagereq type="default">rubygem-googleauth-doc</packagereq>
+      <packagereq type="default">rubygem-google-cloud-env-doc</packagereq>
```

Comment:
is this change intentional?

Reply:
One of the scripts did it, I'll remove the change though